CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Change Log
 ==========
 
+0.5.2
+------
+
+* `uses_south` flag in DjangoApp hook is renamed to `has_migrations` to be
+  compliant with Django 1.8+, which doesn't need South for migrations.
```

Comment:
Backwards-incompatible changes should be accompanied by a minor version number bump (major if we were post 1.0)